src/components/Buttons/LinkButton.tsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import Link from "next/link";
+import styled from "styled-components";
+import Button from "@mui/material/Button";
+
+const UnstyledLink = styled(Link)`
+    text-decoration: none;
+    display: contents;
+`;
+
+const LinkButton: React.FC<{ href: string }> = (props) => {
```

Comment:
can the `{href: string}` be extracted into a Props interface?

Reply:
Done, realised I hadn't actually implemented `LinkButton` so have done that now!